src/index.js

```diff
@@ -38,7 +38,128 @@ module.exports = function ({ types: t }) {
             return otherBeforeEach.indexOf(b) === -1;
           });
         }
-      }
+      },
+      CallExpression(path) {
```

Comment:
May be better to move inside the updating visitor in mine PR. Depends on who's get in first 😆